pyproject.toml

```diff
@@ -27,8 +27,8 @@ classifiers = [
     "Topic :: Scientific/Engineering :: GIS",
 ]
 dependencies = [
-    "titiler.core>=0.11.6,<0.12",
-    "titiler.mosaic>=0.11.6,<0.12",
+    "titiler.core>=0.11.7,<0.12",
+    "titiler.mosaic>=0.11.7,<0.12",
```

Comment:
`0.11.7` is the release where we added customizable template not `0.11.6` 🤦 

ref https://github.com/stac-utils/titiler-pgstac/pull/94